app/Console/Commands/AddFirstLibrariand.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\User;
+use Illuminate\Console\Command;
+
+class AddFirstLibrariand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'librarian:add';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Add the first librarian to the system';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        $name = $this->ask('Enter librarian name');
+        $username = $this->ask('Enter librarian username');
+        $email = $this->ask('Enter librarian email');
+        $jmbg = $this->ask('Enter librarian jmbg');
+        $password = $this->ask('Enter librarian password min 8 and max 16 characters');
+
+        // ... ask for last name, email, username, jmbg and validate them
+
+        $librarian = new User;
+        $librarian->name = $name;
+        $librarian->username = $username;
+        $librarian->email = $email;
+        $librarian->password = $password;
+        $librarian->jmbg = $jmbg;
+        $librarian->user_type = 'librarian';
```

Comment:
In the user class, add CONSTS for the user types, like:

```
public const USER_TYPE_STUDENT  = 'student';
public constUSER_TYPE_LIBRARIAN  = 'librarian';
```

and then instead of using a string to refrence an enum field, use the const value from the class like this:
`$librarian->user_type = User::USER_TYPE_LIBRARIAN`

This is the recommended approach for any enum/dictionary values, to avoid mistakes in spelling while working on them, and it allows smaller code changes in case we decide to change the value of the enum from lets say librarian to Librarian (first letter capitalized)

Reply:
Constant variables created